### DISCUSSION TITLE
Using with Canvas

### DISCUSSION THREAD

**Discussion by biscotty666** [mdelobelle/metadatamenu#434] [Q&A]
I'm trying to understand how to use the canvas fields. They seem intriguing, but how do I actually use them. The documentation describes the possibilities, but in canvases I have no metadata-menu options in the dropdowns. How do I apply a fileClass to a canvas?

Thanks in advance.

**Comment by mdelobelle** [comment]
Hi you can't apply a fileclass to a canvas. You can find demo videos here
https://youtu.be/7oaau8ijVUA
https://youtu.be/G47AYkmoKJs
https://www.youtube.com/watch?v=6HZ3Cc10HS8&t=2160s
